controller/src/board/wlan.py

```diff
@@ -0,0 +1,105 @@
+# wlan.py
+import time
+from .credentials import WLANCredentials
+import os
+
+if "rp2" in os.uname():
```

Comment:
I cannot call this method from a windows computer.  

`os.name` works for me and performs the same function.

Reply:
Oh wow, yeah it looks like that call isn't supported on Windows. I'll try to find another way to do this. I essentially just want to find out whether or not to load stubs in place of the micropython libraries. Maybe I need to re-think the way I'm doing my unit testing in general.

I'll look for another way to do this in the meantime.